pages/media/index.vue

```diff
@@ -51,6 +53,7 @@
         script: [
           { src: `https://code.jquery.com/jquery-${this.JQUERY_VERSION}.min.js` },
           { src: `https://code.jquery.com/ui/${this.JQUERY_UI_VERSION}/jquery-ui.min.js` },
+          { src: this.dashRequired ? `https://unpkg.com/dashjs@${this.DASHJS_VERSION}/dist/dash.all.min.js` : '' },
```

Comment:
This results in an empty <src> tag being added to the markup.
Try this instead:
```
        script: [
          { src: `https://code.jquery.com/jquery-${this.JQUERY_VERSION}.min.js` },
          { src: `https://code.jquery.com/ui/${this.JQUERY_UI_VERSION}/jquery-ui.min.js` },
          { src: `https://unpkg.com/@europeana/media-player@${this.MEDIA_PLAYER_VERSION}/dist/europeana-media-player.min.js` }
        ].concat(this.dashRequired ? [
          { src: `https://unpkg.com/dashjs@${this.DASHJS_VERSION}/dist/dash.all.min.js` }
        ] : [])
```

Reply:
thanks I knew there was better way of doing this, just couldn't think of it